modules/opensimplex/doc_classes/NoiseTexture.xml

```diff
@@ -31,6 +31,9 @@
 		<member name="noise" type="OpenSimplexNoise" setter="set_noise" getter="get_noise">
 			The [OpenSimplexNoise] instance used to generate the noise.
 		</member>
+    <member name="noise_offset" type="Vector2" setter="set_noise_offset" getter="get_noise_offset">
+			An offset used to specify the noise space coordinate of the top left corner of the generated noise. Defaults to (0, 0).
```

Comment:
You don't need to add the default value in the description. Instead, `--doctool` should add the default value for you in an XML attribute. See [Contributing to the class reference](https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html) in the documentation.

```suggestion
			An offset used to specify the noise space coordinate of the top left corner of the generated noise.
```